chapters/annotations.tex

```diff
@@ -632,27 +632,42 @@ \section{Simulations}\label{annotations-for-simulations}\label{simulations}
 
 
 \section{Usage Restrictions}\label{usage-restrictions}
-\subsection{Single Use of Class}\label{annotation-for-single-use-of-class}\label{single-use-of-class}
 
-For state machines it is useful to have single instances of local classes.
-To this end a class may use the annotation:
-\begin{lstlisting}[language=modelica]
-/*literal*/ constant Boolean singleInstance;
-\end{lstlisting}%
-\annotationindex{singleInstance}
+The annotations listed below are used to restrict the ways in which classes may be used.
+\begin{center}
+\begin{tabular}{l|l l}
+\hline
+\tablehead{Annotation} & \tablehead{Description} & \tablehead{Details}\\
+\hline
+\hline
+{\lstinline!singleInstance!} & Allow at most one instance & \Cref{modelica:singleInstance}\\
+{\lstinline!mustBeConnected!} & Connector must be connected at least once & \Cref{modelica:mustBeConnected}\\
+{\lstinline!mayOnlyConnectOnce!} & Connector may at most be connected once & \Cref{modelica:mayOnlyConnectOnce}\\
```

Comment:
I don't think these two do not belong in a section described as "restrict the ways in which classes may be used", as it is placed on the connector components in the class - not on the class itself (in contrast to singleInstance).

Either restore the previous separate "connection restriction" section, or possibly have a more generic heading.
Note that "obsolete" may possible be seen as a restriction on how classes are used.

Reply:
Yes, seems ok.